ssl/t1_lib.c

```diff
@@ -1062,6 +1062,30 @@ int tls_check_sigalg_curve(const SSL *s, int curve)
 }
 #endif
 
+/*
+ * Return the number of security bits for the signature algorithm, or 0 on
+ * error.
+ */
+static int sigalg_security_bits(const SIGALG_LOOKUP *lu)
+{
+    const EVP_MD *md = NULL;
+    int secbits = 0;
+
+    if (!tls1_lookup_md(lu, &md))
+        return 0;
+    if (md != NULL)
+    {
+        /* Security bits: half digest bits */
+        secbits = EVP_MD_size(md) * 4;
+    } else {
+        if (lu->sigalg == TLSEXT_SIGALG_ed25519)
+            secbits = 128;
+        else if (lu->sigalg == TLSEXT_SIGALG_ed448)
+            secbits = 224;
```

Comment:
Is there a reference for these values?

Reply:
https://tools.ietf.org/html/rfc8032#section-8.5

Reply:
Yes, see ecx_meth.c line 21 and 27. I guess I could move the define to some public header.

Reply:
Thanks. Might be worth including the reference @kaduk gave in a comment here (optionally also in ecx_meth.c).

Reply:
I agree adding the reference on both files is a good addition!